core/domain/exp_domain.py

```diff
@@ -1342,6 +1342,24 @@ def validate(self, exp_param_specs_dict, allow_null_interaction):
         elif self.interaction.id is not None:
             self.interaction.validate(exp_param_specs_dict)
 
+    def get_training_data(self):
```

Comment:
@seanlip Done. Didn't really know how I could test this. I sort of type checked every element in training_data.